### DISCUSSION TITLE
Import PopoverStateReturn

### DISCUSSION THREAD

**Discussion by kcubero27** [twilio-labs/paste#2465] [Q&A] [answered]
Good morning,

We have realised that the PopoverContainer and usePopoverState don't use the same interface. This creates some problems when trying to use the interface PopoverStateReturn in other projects. In our case, we have created a component on top of it using a lot of this logic but we are not able to make it run as TypeScript complains about different types.

How can we fix help this?

In case you want an example, please DM. 

Thank you for your help!

**Comment by kate-porter** [comment]
Hey @kcubero27 - thanks for reaching out about this. I'm sharing with the team and we'll get back to you soon.

**Comment by kcubero27** [comment, marked as answer]
I believe the issue was related to that we hadn't linked twilio-paste/core using npm link. Sorry for the inconvenience 🙏🏼

**Reply by kate-porter** [reply]
No problem, thanks for letting us know!
